students/rgpag/lesson03/list_lab.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env
+
+#series1
+series1 = ["Apples","Pears","Oranges","Peaches"]
+def series_1():
+	print(series1)
+	response = [input("Please enter another fruit > ")]
+	series1=series1+response
```

Comment:
Redefining this name in the local namespace of the function is causing an `UnboundLocalError` in the print call on line 6. I would suggest naming the global variable something like `fruits` to avoid this